core/src/main/java/com/linecorp/armeria/common/util/EventLoopThreadFactory.java

```diff
@@ -19,48 +19,58 @@
 import static java.util.Objects.requireNonNull;
 
 import java.util.concurrent.ThreadFactory;
+import java.util.function.Function;
 
 import javax.annotation.Nullable;
 
-import io.netty.util.concurrent.DefaultThreadFactory;
-
 /**
  * {@link ThreadFactory} that creates event loop threads.
  *
  * @see EventLoopGroups
+ *
+ * @deprecated Use {@link ThreadFactories#builderForEventLoops(String)}.
  */
-public final class EventLoopThreadFactory implements ThreadFactory {
-
-    // Note that we did not extend DefaultThreadFactory directly to hide it from the class hierarchy.
-    private final ThreadFactory delegate;
-
+@Deprecated
+public final class EventLoopThreadFactory extends AbstractThreadFactory {
     /**
      * Creates a new factory that creates a non-daemon and normal-priority thread.
      *
      * @param threadNamePrefix the prefix of the names of the threads created by this factory.
+     *
+     * @deprecated Use {@link ThreadFactories#builderForEventLoops(String)}.
      */
+    @Deprecated
```

Comment:
We can now remove the `@[Dd]eprecated` for all methods in this class because the class has been deprecated.

Reply:
Removed all `@[Dd]precated` inside of `EventLoopThreadFactory`